### DISCUSSION TITLE
Alert and result set

### DISCUSSION THREAD

**Discussion by stphnlwlsh** [datalust/seq-tickets#1323] [General]
Not sure how or if this can be accomplished, but we log lots of events to SEQ and then want email reports back for them. Initially I was using the Digest Email to get a list of events into an email, but that becomes a single point of creation because only admins can create new instances of the Digest Email.

Since everyone on our instance can create dashboards, I was hoping to create a few standard templates for everyone to make use of when they go to create their various reports (via dashboard alerts).

The problem that I'm encountering is that if I have a signal that is giving off 15 events (i.e. exceptions of http 500 error for example), I want to alert with the _@Message_, but I don't see a way to do that currently. The alert only ever sends out a single event indicating that there's more to see, but you have to click over here to the dashboard and then view your results. This  isn't ideal as our SEQ install is behind a corporate firewall which is inaccessible from mobile devices. As mentioned above, I _CAN_ accomplish this with a digest email, but not everyone needs to be an admin.

Am I holding it wrong? Is what I'm asking making sense and possible ?

Thanks!

**Comment by nblumhardt** [comment]
Hi @stphnlwlsh, thanks for dropping by!

What you're looking for makes sense, and it's one of the features coming in 2021.3's new alerting improvements (#1126), called "Include contributing events":

![image](https://user-images.githubusercontent.com/342712/133694241-79de4acd-6aa2-44d0-9323-8579bbb6764a.png)

The feature is in the current 2021.3 previews, but output apps like Slack and Email aren't updated to work with it yet, so it might not be time to upgrade (notifications won't be formatted nicely enough for a wide audience).

We're aiming to publish weekly updates, though, and should have the final version ready within a month or so 🤞 .

Best regards,
Nick

**Comment by stphnlwlsh** [comment]
Excellent! Thanks very much for adding this in. It's much needed to get some of the admin duty off my plate 😊
